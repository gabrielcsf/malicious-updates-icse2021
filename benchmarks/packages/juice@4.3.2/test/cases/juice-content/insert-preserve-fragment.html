<div>here</div>
<div>not here</div>
<p></p>
