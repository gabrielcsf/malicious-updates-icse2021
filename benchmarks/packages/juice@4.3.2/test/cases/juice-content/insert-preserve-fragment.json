{
    "preserveFontFaces": true,
    "preserveMediaQueries": true,
    "insertPreservedExtraCss": "div",
    "extraCss": "@font-face { font-family: 'Extra'; }"
}
