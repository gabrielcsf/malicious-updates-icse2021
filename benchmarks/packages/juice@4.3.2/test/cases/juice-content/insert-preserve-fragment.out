<div>here<style>
@font-face {
  font-family: 'Extra';}
</style></div>
<div>not here</div>
<p></p>
